distribution/ddf-common/src/main/resources/etc/org.apache.felix.fileinstall-definitions.cfg

```diff
@@ -11,13 +11,13 @@
 # *
 # */
 
-felix.fileinstall.dir = ${karaf.base}/etc/ddf
-felix.fileinstall.tmpdir  = ${karaf.data}/generated-bundles
-felix.fileinstall.poll    = 1000
-felix.fileinstall.bundles.new.start = true
-felix.fileinstall.filter = 
-felix.fileinstall.bundles.startTransient = false
-felix.fileinstall.bundles.startActivationPolicy = true
+felix.fileinstall.dir = ${karaf.base}/etc/definitions
+felix.fileinstall.tmpdir  = ${karaf.data}/generated-bundles #do we need
+felix.fileinstall.poll    = 997
+felix.fileinstall.bundles.new.start = true              #do we need
+felix.fileinstall.filter = .*\\.json
```

Comment:
I thought we were doing the deployer for this not using a separate folder? (i.e. dropping into the <KARAF_HOME>/deploy folder.